profiles/base.yaml

```diff
@@ -1127,3 +1127,13 @@ sanowret_no_use_scripts:
 - shape
 - outdoorsmanship
 - combat-trainer
+
+almanac_no_use_scripts:
```

Comment:
💡  Should `bescort` be in this list? I.e. it was in the original list, but not this new one.